lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -101,7 +106,16 @@ export class PublicClientApplication {
      * containing data from the server (returned with a null or non-blocking error).
      */
     handleRedirectCallback(authCallback: AuthCallback): void {
-        throw new Error("Method not implemented.");
+        if(!authCallback) {
+            throw BrowserConfigurationAuthError.createInvalidCallbackObjectError(authCallback);
+        }
+
+        this.authCallback = authCallback;
+        const urlHash = window.location.hash;
```

Comment:
ES6 tip: `const { location: { hash } } = window;`

Reply:
@jasonnutter  is this null safe ?